OWNERS

```diff
@@ -4,10 +4,16 @@ reviewers:
   - sttts
   - deads2k
   - slaskawi
+  - thejasn
+  - TrilokGeer
+  - swghosh
```

Comment:
I like to be appended as optional reviewer : ) I'm OpenShift Auth QE lead who will also test cert-manager, like to review when I can. If not appropriate to be added, nvm that is fine for me.

Reply:
Will raise a separate PR adding you as reviewer.